common/internal_model/src/main/scala/uk/ac/wellcome/models/work/internal/Image.scala

```diff
@@ -68,13 +68,6 @@ object ImageState {
 
   case class Initial(
     sourceIdentifier: SourceIdentifier,
-  ) extends ImageState {
-    type SourceDataState = DataState.Unidentified
-    type TransitionArgs = Unit
-  }
-
-  case class Identified(
-    sourceIdentifier: SourceIdentifier,
     canonicalId: String
   ) extends ImageState {
     type SourceDataState = DataState.Identified
```

Comment:
Can we now remove `SourceDataState` entirely (as it's always `Identified`)? Maybe best done in a subsequent PR, I'd be happy to do that if you want

Reply:
Ah yeah makes sense, I didn't spot that. Best in a different PR though, this one is big enough. I can do that if you want

Reply:
Yeah go for it - I think it will get rid of the most unpleasant aspects of the image state stuff